include/mesos/allocator/allocator.hpp

```diff
@@ -28,6 +28,8 @@
 
 #include <mesos/quota/quota.hpp>
 
+#include <master/constants.hpp>
```

Comment:
The definitions in the public allocator interface must not depend on the internal ones. 
I.e. the headers in `include/` should be usable without ones from `src/`.

Is it possible to define these constants here, either in this header or in a some other header (in the worst case, in a new one)?

Reply:
So the flags defined for master will depend on `include/allocator/allocator.hpp` header (instead of the more generic `constants.hpp`? Is that ok?